extension/indexer/accepted_subscriber.go

```diff
@@ -0,0 +1,31 @@
+// Copyright (C) 2024, Ava Labs, Inc. All rights reserved.
+// See the file LICENSE for licensing terms.
+
+package indexer
+
+import (
+	"context"
+
+	"github.com/ava-labs/hypersdk/chain"
+)
+
+type AcceptedSubscriber interface {
+	Accepted(ctx context.Context, blk *chain.StatelessBlock) error
```

Comment:
I don't think we need this context + in the other tx subscriber interface currently... we could leave it in since I think it's reasonable that you would want a context though (say if you were archiving into s3 or something).

Reply:
Ya I think this is reasonable to leave in as part of the interface